settings.gradle

```diff
@@ -3,4 +3,5 @@ include ':server_extensions_extra'
 include ':server_extensions_test'
 include ':currencies'
 include ':batm_ssh_tunnel'
+include 'operators_sample_website'
```

Comment:
Project name should start with colon, it works for `include` key word but that's all. So please stick to it